electrum/gui/kivy/uix/dialogs/request_dialog.py

```diff
@@ -159,25 +179,47 @@ def on_mode(self, instance, x):
             self.ids.qr.opacity = 1
         else:
             self.ids.qr.opacity = 0
```

Comment:
changing the qr opacity is not needed anymore, I guess.

Reply:
right. removed in follow-up https://github.com/spesmilo/electrum/commit/13570a465c0c017803f45cb4ad96f2b349ffc2bb